demos/checkbox.html

```diff
@@ -201,6 +202,10 @@
         document.getElementById('make-ind').addEventListener('click', function() {
           checkboxInstance.indeterminate = true;
         });
+
+        document.getElementById('toggle-disabled').addEventListener('click', function() {
```

Comment:
@amsheehan Hmm I expected it would be added by the API.. I wonder if there is a setter that adds the `mdc-checkbox--disabled` class to the root element without adding it explicitly.

Otherwise, of course, I will address it by adding the following:
`checkboxInstance .addClass('mdc-checkbox--disabled');`